app/helpers/test_passages_helper.rb

```diff
@@ -0,0 +1,17 @@
+module TestPassagesHelper
+  def test_result(test_passage)
+    if test_passage.passed?
+      return 'You have passed the test!'
```

Comment:
`return` не используется в руби для последней строки исполняемого кода метода
последняя строка возвращаетя по умолчанию
```suggestion
      'You have passed the test!'
```
аналогично в остальных случаях